nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/resolver/RegexNamespaceResolver.java

```diff
@@ -34,12 +34,12 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
-public class RegexClusterResolver implements ClusterResolver {
+public class RegexNamespaceResolver implements ClusterResolver {
```

Comment:
Renaming the class was a good idea. Could you please adjust the interface name too?